joomla/joomla-cms
# [5.0] Update deleted files and folders to changes from #40293

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the lists of files and folders to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 5.0-dev branch.

Currently there are only changes caused by PR #40293 for removing the "symfony/polyfill-php72" dependency.

### Testing Instructions

Code review.

Or if you want to make a real test, update the last 4.4 nightly build to the last 5.0 nightly build to get the actual result, and update the last 4.4 nightly build to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

The files and folders of the "symfony/polyfill-php72" dependency are still present after updating from the last 4.4 nightly build.

### Expected result AFTER applying this Pull Request

The files and folders of the "symfony/polyfill-php72" dependency have been deleted after updating from the last 4.4 nightly build, like it is done already without this PR here with the symfony polyfills for the other PHP versions.

### Link to documentations
Please select:
